### DISCUSSION TITLE
Export/Import of customizations

### DISCUSSION THREAD

**Discussion by profucius** [Mahmud0808/Iconify#194] [Q&A]
Hi, I have a question and did not know where to find the answer. I'm using Iconify on my phone. I had everything setup the way I like it, but recently had to format my phone and restore my apps from a backup I made using SwiftBackup (total replacement for TitaniumBackup).

I noticed that when I restored Iconify's data, it did not carry over all the customizations I made previously. I also scoured the app for an export/import feature, but didn't find one. Am I overlooking it somewhere, or does the app not yet have this feature?

If not yet, I propose adding this feature. You never know when suddenly you'll need to reformat your phone OS, and being able to restore all the hard work in applying customizations through Iconify would be a huge help to reduce the work involved.

**Comment by Mahmud0808** [comment]
Will keep in mind. Thanks for the suggestion.
